salesforce/client.bal

```diff
@@ -69,6 +65,250 @@ public isolated client class Client {
         }
     }
 
+    //Describe SObjects
+    # Get metadata for your organization.
+    #
+    # + return - `OrganizationMetadata` record if successful or else `sfdc:Error`
+    @display {label: "Get Organization Metadata"}
+    isolated remote function getOrganizationMetaData() returns @display {label: "Organization Metadata"}
+                                                    OrganizationMetadata|error {
+        string path = utils:prepareUrl([API_BASE_PATH, SOBJECTS]);
+        return check self.salesforceClient->get(path);
+    }
+
+    # Describes the individual metadata for the specified object.
+    #
+    # + sobjectName - sObject name
+    # + return - `SObjectBasicInfo` record if successful or else `sfdc:Error`
+    @display {label: "Get sObject Basic Information"}
+    isolated remote function getBasicInfo(@display {label: "sObject Name"} string sobjectName)
+                                                returns @display {label: "sObject Basic Information"}
+                                                SObjectBasicInfo|error {
+        string path = utils:prepareUrl([API_BASE_PATH, SOBJECTS, sobjectName]);
+        return check self.salesforceClient->get(path);
+    }
+
+    # Completely describes the individual metadata at all levels for the specified object. Can be used to retrieve
+    # the fields, URLs, and child relationships.
+    #
+    # + sObjectName - sObject name value
+    # + return - `SObjectMetaData` record if successful or else `sfdc:Error`
+    @display {label: "Get sObject Description"}
+    isolated remote function describe(@display {label: "sObject Name"} string sObjectName)
+                                            returns @display {label: "sObject Metadata"} SObjectMetaData|error {
+
+        string path = utils:prepareUrl([API_BASE_PATH, SOBJECTS, sObjectName, DESCRIBE]);
+        return check self.salesforceClient->get(path);
+    }
+
+    # Query for actions displayed in the UI, given a user, a context, device format, and a record ID.
+    #
+    # + return - `SObjectBasicInfo` record if successful or else `sfdc:Error`
+    @display {label: "Get sObject Platform Action"}
+    isolated remote function getPlatformAction() returns @display {label: "sObject Basic Information"}
+                                                SObjectBasicInfo|error {
+        string path = utils:prepareUrl([API_BASE_PATH, SOBJECTS, PLATFORM_ACTION]);
+        return check self.salesforceClient->get(path);
+    }
+
+    //Describe Organization
+    # Lists summary details about each REST API version available.
+    #
+    # + return - List of `Version` if successful. Else, the occured Error.
```

Comment:
This should be error

Reply:
Fixed inhttps://github.com/ballerina-platform/module-ballerinax-sfdc/commit/666dfedf90963e702c3e32712fa8e2e41f087f2ehttps://github.com/ballerina-platform/module-ballerinax-sfdc/commit/666dfedf90963e702c3e32712fa8e2e41f087f2e